.packit.yaml

```diff
@@ -0,0 +1,49 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: foreman-installer.spec
+
+# add or remove files that should be synced
+files_to_sync:
+  - foreman-installer.spec
+  - .packit.yaml
```

Comment:
We have this all over the place and it's not needed. Let's at least start to keep it clean with new configs.
```suggestion
```